dashboard/src/components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+import "./LoadingSpinner.css";
+
+const LoaderSpinner: React.SFC<{}> = () => {
```

Comment:
Nice find the `SFC` (apart from the ugly acronym :P )

Reply:
This can be replaced by `React.StatelessComponent` but I thought that Sevilla Futbol Cub deserved a mention :trollface: